### PR TITLE
Fix a questionable version check.

### PR DESCRIPTION
**Pull request by mdickinson**
`TabDragObject.get_pixmap` switches on `QtGui.qt_api` to decide whether to use `widget.grab()` (available only in Qt 5) or `QPixmap.grabWidget(widget)` (available only in Qt 4).

But this seems like the wrong check, and will likely do the wrong thing for PySide 2. We're not interested in the pyqt version; we're interested in the Qt version.

This PR fixes that, and tidies up some nearby code in passing.

### CODE REVIEW

File: pyface/ui/qt4/tasks/split_editor_area_pane.py [hunk 1]
```diff
@@ -939,17 +939,17 @@ def get_pixmap(self):
         painter.drawPrimitive(QtGui.QStyle.PE_FrameTabBarBase, optTabBase)
 
         # region of active tab
-        if QtGui.qt_api=='pyqt5':
-            pixmap1 = tabBar.grab()
-        else: # pyqt4 (grabWidget was removed in pyqt5)
+        if QtCore.QT_VERSION >= 0x050000:
+            pixmap1 = tabBar.grab(tab_rect)
```
Comment (mdickinson) [inline pyface/ui/qt4/tasks/split_editor_area_pane.py]: It looks to me as though the missing `tab_rect` in the original code was unintentional: I can't figure out why we'd want it for Qt4 but not for Qt5.

**Comment by mdickinson** [comment]
Also, our default assumption should be that Qt 6 (should such a thing ever arise) will have the same behaviour as Qt 5 rather than Qt 4.

**Comment by mdickinson** [comment]
And yes, this should have a regression test.

**Comment by mdickinson** [comment]
Gah. This won't work on PySide either: there's apparently no `QtCore.QT_VERSION` available on (old) PySide. Don't know whether it exists on PySide 2.

**Comment by mdickinson** [comment]
> Gah. This won't work on PySide either

Fixed.

**Comment by mdickinson** [comment]
Should be updated (again) to use `is_qt4`.

File: pyface/ui/qt4/tasks/split_editor_area_pane.py [hunk 2]
```diff
@@ -939,18 +939,18 @@ def get_pixmap(self):
...
-        if QtGui.qt_api=='pyqt5':
-            pixmap1 = tabBar.grab()
-        else: # pyqt4 (grabWidget was removed in pyqt5)
+        if is_qt4:  # grab wasn't introduced until Qt5
             pixmap1 = QtGui.QPixmap.grabWidget(tabBar, tab_rect)
+        else:
+            pixmap1 = tabBar.grab(tab_rect)
 
-        painter.drawPixmap(0,0,pixmap1) #tab_rect.topLeft(), pixmap1)
+        painter.drawPixmap(0, 0, pixmap1)  # tab_rect.topLeft(), pixmap1)
```
Comment (rahulporuri) [inline pyface/ui/qt4/tasks/split_editor_area_pane.py]: What is the significance of the commented out code? Can we throw it out? (I understand that this might have been a drive-by change).

Comment (mdickinson) [reply pyface/ui/qt4/tasks/split_editor_area_pane.py]: Yes, I suspect we should throw it out, unless anyone remembers why it was changed in the first place.

Comment (mdickinson) [inline pyface/ui/qt4/tasks/split_editor_area_pane.py]:
```suggestion
        painter.drawPixmap(0, 0, pixmap1)
```

**Comment by corranwebster** [comment]
LGTM
